astropy/io/ascii/cds.py

```diff
@@ -5,25 +5,66 @@
   Classes to read CDS / Vizier table format
 
 :Copyright: Smithsonian Astrophysical Observatory (2011)
-:Author: Tom Aldcroft (aldcroft@head.cfa.harvard.edu)
+:Author: Tom Aldcroft (aldcroft@head.cfa.harvard.edu), \
+         Suyog Garg (suyog7130@gmail.com)
 """
 
 
+from io import StringIO
```

Comment:
I know I suggested you move this up since it is a standard library import (which come first according to PEP8), but `os` and `math` are also standard library imports; so just move this line after `import numpy as np`.